src/kernel/chainparams.cpp

```diff
@@ -395,10 +395,10 @@ class CTestNet4Params : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from RPC: getchaintxstats 4096 000000005be348057db991fa5d89fe7c4695b667cfb311391a8db374b6f681fd
-            .nTime    = 1723651702,
-            .tx_count = 757229,
-            .dTxRate  = 0.01570402633472492,
+            // Data from RPC: getchaintxstats 4096 00000000798bd720800263b69bee01aae57ea376596415a04423da29f000c367
```

Comment:
In 570191a758bc326ae0f0a479b72759f962c0db12 "[kernel] update chainTxData for v29"

Comment needs to be updated for the block that these stats are for.

Reply:
fixed